src/components/Chip/Chip.tsx

```diff
@@ -18,85 +17,54 @@ export interface Props {
   moreInfoComponentShowOn?: DisplayMoreInfo;
   style?: React.CSSProperties;
   theme?: any;
-  onRemove?(event: React.FormEvent<HTMLElement>): void;
-  onClick?(event: React.FormEvent<HTMLElement>): void;
+  onRemove?(event: any): void;
```

Comment:
Chip onremove and onclick - event type changed to any - revert it and check

Reply:
resolved